### PR TITLE
drop support for Kubernetes 1.9 which is now EOL

### PR DESCRIPTION
**Pull request by kdomanski**
```release-note
The EOL Kubernetes 1.9 is no longer supported
```

### CODE REVIEW

File: config/kubermatic/static/master/updates.yaml [hunk 1]
```diff
@@ -2,14 +2,10 @@ updates:
 # ======= 1.8 =======
 # Allow to next minor release
 - from: 1.8.*
-  to: 1.9.0
```
Comment (alvaroaleman) [inline config/kubermatic/static/master/updates.yaml]: Can we just jump minor versions? I am not completely sure there WRT the apiserver

Comment (kdomanski) [reply config/kubermatic/static/master/updates.yaml]: Done.

File: config/kubermatic/static/master/updates.yaml [hunk 2]
```diff
@@ -6,10 +6,6 @@ updates:
   automatic: true
 
 # ======= 1.9 =======
-# Allow to change to any patch version
-- from: 1.9.*
-  to: 1.9.*
-  automatic: false
 # Allow to next minor release
 - from: 1.9.*
   to: 1.10.1
```
Comment (mrIncompetent) [inline config/kubermatic/static/master/updates.yaml:11]: Should we make this update `automatic: true` ?

Comment (mrIncompetent) [inline config/kubermatic/static/master/updates.yaml:11]: Should we make this update `automatic: true` ?

Comment (kdomanski) [reply config/kubermatic/static/master/updates.yaml:11]: I think we should leave it as it is, but mention in the release notes that support is pulled and the clusters should be upgraded ASAP.

**Review by mrIncompetent — APPROVED** [review]

**Review by alvaroaleman — APPROVED** [review]

**Comment by mrIncompetent** [comment]
Why should this be cherry picked into v2.8 ?

**Comment by kdomanski** [comment]
Because v1.9 is EOL as of v2.8 and we should have removed it at the same time as we added v1.12

**Comment by mrIncompetent** [comment]
> Because v1.9 is EOL as of v2.8 and we should have removed it at the same time as we added v1.12

Kubermatic v2.8 supports kubernetes 1.9-1.12.
We published v2.8 with those versions and removing them is a breaking change.

Just because Kubernetes 1.9 is EOL, does not mean kubermatic is suddenly not able to manage it anymore.

**Comment by kdomanski** [comment]
Fair enough.

**Pull request merged**